src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json

```diff
@@ -14,13 +14,29 @@
                 "name": "OsConfigPolicy",
                 "version": "1.0.0",
                 "contentType": "Custom",
-                "contentUri": "<SAS token URL for the artifacts package>",
-                "contentHash": "<file hash for the artifacts package>",
+                "contentUri": "https://gcpoc88.blob.core.windows.net/blobforosconfig/LinuxSshServerSecurityBaselinePolicy.zip?sp=r&st=2024-02-20T19:49:37Z&se=2034-02-21T03:49:37Z&sv=2022-11-02&sr=b&sig=fiZZf8ML276R7PLkpGvYgwHZKIB9e67WU%2BcFAhs%2BACE%3D",
```

Comment:
Is this intended? This might cause so noise by credential scanners (reacting to a SAS token).